utils.ml

```diff
@@ -39,7 +39,8 @@ let read_file path =
 
 open Yojson.Basic.Util
 
```

Comment:
What do you think of adding `utils.mli`? You can keep better track of unused functions and also add some documentation.